app/controllers/admin/users_controller.rb

```diff
@@ -1,5 +1,20 @@
 module Admin
   class UsersController < ApplicationController
+    def update
+      @user = User.find(params[:id])
+      if @user.editable_by?(current_user)
+        if @user.admin?
+          @user.set_member
+          flash[:notice] = "User Demoted to Member!"
+          redirect_to admin_neighborhoods_path
+        else
+          @user.set_admin
+          flash[:notice] = "User Promoted to Admin!"
+          redirect_to admin_neighborhoods_path
+        end
+      end
+    end
```

Comment:
If we do in this way, I think we can't have admins change any other features of users. Do you think this means we're losing to much flexibility? (I'm thinking like, someone has an inappropriate profile pic - what if we want an admin to delete that?)